### ISSUE TITLE
Feature request : allow the seeds directory config param to accept an array of directory

### ISSUE THREAD

**Issue by mdartic** [knex/knex#3951]
# Feature discussion / request

1. Explain what is your use case

I would like to seed my database with several directories of seeds.

My project using Knex is quite "generic", and I would like to add, depending the instance, some directories of seeds, and explain to knex - via the knexfile config file - in which directories it has to find the seeds files.

2. Explain what kind of feature would support this

--

3. Give some API proposal, how the feature should work

I imagine that the config file could look like this :

```js
  seeds: {
    directory: [
       './seeds/myFirstFolder/',
       './seeds/mySecondFolder/',
       './seeds/myThirdFolder/'
    ]
  }
```

**Comment by kibertoad** [comment]
@mdartic I assume this was implemented in #3967?

**Comment by mdartic** [comment]
Yes, I think it's already implemented [here](https://github.com/knex/knex/pull/3967/files#diff-80fa106f911725b4cecd1eed619be2846298ae05e8e41fb504eb0c252d505904R154-R156).

We can close this issue indeed.
